service/storage_proxy.cc

```diff
@@ -2795,6 +2798,9 @@ void storage_proxy::send_to_live_endpoints(storage_proxy::response_id_type respo
             std::optional<sstring> msg;
             try {
                 std::rethrow_exception(eptr);
+            } catch (replica::rate_limit_exception&) {
+                // There might be a lot of those, so ignore
```

Comment:
Should we count them?

Reply:
Yeah, if we are counting timeouts then probably it's a good idea to count rate limit exceptions, too. I'll add metrics, one for reads and one for writes.

Reply:
Added replica- and coordinator-side metrics. The coordinator actually has two metrics, separated by a label, one counts cases where the coordinator rejected and the other counts the cases where replicas made the decision to reject.